CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
 include(GitExternal)
 
 set(VERSION_MAJOR "0")
-set(VERSION_MINOR "4")
+set(VERSION_MINOR "5")
 set(VERSION_PATCH "0")
-set(VERSION_ABI 3) # OFF or ABI version
```

Comment:
BTW. Today was the last day for release. The commit to fix camera in volume is done but the question is will we include this commit too ?